plugins/BEdita/Core/src/Model/Table/ObjectTypesTable.php

```diff
@@ -459,4 +460,26 @@ protected function findByRelation(Query $query, array $options = [])
         // Everything is said and done by now. Fingers crossed!
         return $query->where($conditionsBuilder);
     }
+
+    /**
+     * Finder to get object type starting from object id or uname.
+     *
+     * @param \Cake\ORM\Query $query Query object.
+     * @param array $options Additional options. The `id` key is required.
+     * @return \Cake\ORM\Query
+     * @throws \LogicException When missing required parameters.
+     */
+    protected function findObjectId(Query $query, array $options = [])
+    {
+        if (empty($options['id'])) {
+            throw new \LogicException(__d('bedita', 'Missing required parameter "{0}"', 'id'));
```

Comment:
Maybe `BadFilterException` is more appropriate.

Reply:
I had copied that exception from a finder above :smile_cat: 
I'll change it.